atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceTableManipulationIntegrationTest.java

```diff
@@ -175,16 +175,16 @@ public void tableCreationAppearsToSucceedButIsNoop() {
                 .contains(BYTE_ARRAY);
 
         assertThatThrownBy(() -> kvs.get(LOWER_LOWER, ImmutableMap.of(CELL, 2L)))
-                .isInstanceOf(RetryLimitReachedException.class);
+                .hasCauseInstanceOf(RetryLimitReachedException.class);
 
         assertThatThrownBy(() -> kvs.put(LOWER_LOWER, ImmutableMap.of(CELL, SECOND_BYTE_ARRAY), 1))
-                .isInstanceOf(RetryLimitReachedException.class);
+                .hasCauseInstanceOf(RetryLimitReachedException.class);
     }
 
     private void createTablesIgnoringException() {
         assertThatThrownBy(() -> kvs.createTables(TABLES.stream()
                         .collect(Collectors.toMap(x -> x, no -> AtlasDbConstants.GENERIC_TABLE_METADATA))))
-                .isInstanceOf(IllegalStateException.class);
+                .hasCauseInstanceOf(IllegalStateException.class);
```

Comment:
I assume there was a decision here that got changed - these tests should still verify isInstanceOf if I'm not mistaken

Reply:
No - these go through `TimedRunner`, which was throwing a SafeRuntimeException, not the original exception. However, I think I'm going to refactor that because it is less invasive.